### PR TITLE
release: 0.19.0

### PR DESCRIPTION
**Pull request by davidhewitt**
As per #3154 I think we're ready to push 0.19.

I'll put this live on Tuesday evening unless I hear a reason to postpone.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,49 @@ To see unreleased changes, please see the [CHANGELOG on the main branch guide](h
...
+## [0.19.0] - 2023-05-28
+
+### Packaging
+
+- Correct dependency on syn to version 1.0.85 instead of the incorrect version 1.0.56. [#3152](https://github.com/PyO3/pyo3/pull/3152)
+
+### Added
+
+- Support `text_signature` option (and automatically generate signature) for `#[new]` in `#[pymethods]`. [#2980](https://github.com/PyO3/pyo3/pull/2980)
+- Added support for converting to and from Python's `decimal.Decimal` and `rust_decimal::Decimal`. [#3016](https://github.com/PyO3/pyo3/pull/3016)
```
Comment (adamreichold) [inline CHANGELOG.md:22]: I now it is busy work that we should have done when merging these PR, but maybe we can harmonize the tense to present? It hopefully is less work to do it once while preparing the changelog.

Comment (davidhewitt) [reply CHANGELOG.md:22]: Agreed, I'll fixup when pulling in #3188. I was so tired last night in the end 😅

**Comment by davidhewitt** [comment]
Rebased and I'll push shortly.

**Comment by davidhewitt** [comment]
(I also added a section to the migration guide to document the changes to `__traverse__`, as some users may be caught out by the new panic.)

**Comment by adamreichold** [comment]
> (I also added a section to the migration guide to document the changes to `__traverse__`, as some users may be caught out by the new panic.)

Thank you and I think this should have been part of the original PR but I never looked back at the guide after writing that one sentence in the class customization section.

**Pull request merged**